### ISSUE TITLE
New yii2 version widget problems

### ISSUE THREAD

**Issue by gerardguiu** [ruskid/yii2-stripe#24]
Hi! The module It have a problem with the new version of yii2 core (2.0.16). This release producte that the 'stripeForm' not show. Is posible that the problem is related with the widget core? Is weird ...

**Comment by ruskid** [comment]
Hey there, haven't tested it with 2.0.16 yet. Just in case are you sure that you print the form? 

I use this form in some of my projects so if there is a problem it will be fixed with next yii2 update. In the meanwhile, you can take a look!

You know this form is getting depreciated. It is using Stripe js v2 and there is Stripe Elements (v3) released which is nice. Also, Jquery Payment library is depreciated.

If there is anybody who wants to create StripeFormV3 that would be great!

Saludos

**Comment by gerardguiu** [comment]
Yes, the StripeFormV2 will be deprecated in september... Are you analized the new api?
